### PR TITLE
Improve/refactor showImage (Link-as-Image), implement Amenity menu similar to WptMenu (html description, image, etc)

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

**Comment by RZR-UA** [comment]
https://github.com/osmandapp/OsmAnd/issues/19787
https://github.com/osmandapp/OsmAnd/issues/21569

File: OsmAnd/src/net/osmand/plus/track/fragments/ReadDescriptionFragment.java [hunk 1]
```diff
@@ -123,29 +116,6 @@ private void setupToolbar(@NonNull View view) {
...
 	}
 
-	private void setupImage(@NonNull View view) {
-		String imageUrl = getImageUrl();
-		if (imageUrl == null) return;
-
-		AppCompatImageView ivImage = view.findViewById(R.id.main_image);
-		PicassoUtils picasso = PicassoUtils.getPicasso(app);
-		RequestCreator rc = Picasso.get().load(imageUrl);
-		WikivoyageUtils.setupNetworkPolicy(app.getSettings(), rc);
```
Comment (Chumva) [inline OsmAnd/src/net/osmand/plus/track/fragments/ReadDescriptionFragment.java:133]: We don't need it anymore?

Comment (RZR-UA) [reply OsmAnd/src/net/osmand/plus/track/fragments/ReadDescriptionFragment.java:133]: It is got back and tested for on/off, including cleared images cache.

Comment (RZR-UA) [reply OsmAnd/src/net/osmand/plus/track/fragments/ReadDescriptionFragment.java:133]: 38cd377ffb

**Pull request merged**